xen-bugtool

```diff
@@ -1516,6 +1520,38 @@ def multipathd_topology(cap):
 
     return stdout
 
+def snmp_xs_conf(cap):
+    sed_snmp_xs_conf = ['sed',
+    '-e', 's#\(community *= *\).*#\\1********#',
+    '-e', 's#\(authentication_key *= *\).*#\\1********#',
+    '-e', 's#\(privacy_key *= *\).*#\\1********#',
+    '/etc/snmp/snmp.xs.conf']
+    pipe = Popen(sed_snmp_xs_conf, bufsize=1, stdin=PIPE, stdout=PIPE, stderr=dev_null)
+    stdout, _ = pipe.communicate()
+    return stdout
+
+def snmpd_xs_conf(cap):
+    sed_snmpd_xs_conf = ['sed',
+    '-e', 's#\(com2sec\( \+\S\+\)\{2\} \+\).*#\\1********#',
+    '/etc/snmp/snmpd.xs.conf']
+    pipe = Popen(sed_snmpd_xs_conf, bufsize=1, stdin=PIPE, stdout=PIPE, stderr=dev_null)
+    stdout, _ = pipe.communicate()
+    return stdout
+
+def snmpd_conf(cap):
+    sed_snmpd_conf = ['sed',
+    '-e', 's#\(usmUser\( \+\S\+\)\{7\} \+\)\S\+\( \+\S\+ \+\)\S\+\( \+\S\+\)#\\1********\\3********\\4#',
+    '/var/lib/net-snmp/snmpd.conf']
+    pipe = Popen(sed_snmpd_conf, bufsize=1, stdin=PIPE, stdout=PIPE, stderr=dev_null)
+    stdout, _ = pipe.communicate()
+    return stdout
+
+def snmpd(cap):
+    cat_snmpd = ['cat', '/etc/sysconfig/snmpd']
+    pipe = Popen(cat_snmpd, bufsize=1, stdin=PIPE, stdout=PIPE, stderr=dev_null)
+    stdout, _ = pipe.communicate()
+    return stdout
+
```

Comment:
```suggestion
```
Replace this with a simple `file_output(CAP_XENSERVER_CONFIG, [SYSCONFIG_SNMPD])`